unstructured/chunking/title.py

```diff
@@ -104,6 +104,10 @@ class _ByTitleChunkingOptions(ChunkingOptions):
     multipage_sections
         Indicates that page-boundaries should not be respected while chunking, i.e. elements
         appearing on two different pages can appear in the same chunk.
+    combine_text_under_n_chars
```

Comment:
Keep the options in alphabetical order. I think they show up that way in all the other lists and it becomes more important as more are added.